### PR TITLE
Add python samples

### PR DESCRIPTION
**Pull request by FedyuninV**

### CODE REVIEW

File: NeoML/Python/neoml/Blob.py [hunk 1]
```diff
@@ -136,9 +136,10 @@ def asarray(self, copy=False):
             more convenient access to the same data.
             Not copying may be impossible if the blob is in GPU memory.
         """
-        if type(self.math_engine) is MathEngine.GpuMathEngine:
-            copy = True
-        return numpy.array(self._internal, copy=copy)
+        if type(self.math_engine) is MathEngine.CpuMathEngine:
+            return numpy.array(self._internal, copy=copy)
+        cpu_blob = self.copy(MathEngine.default_math_engine())
+        return numpy.array(cpu_blob._internal, copy=copy)
```
Comment (SAngeliuk) [inline NeoML/Python/neoml/Blob.py]: copy=false

File: NeoML/Python/neoml/Dnn/Dnn.py [hunk 2]
```diff
@@ -138,6 +138,21 @@ def layers(self):
         """
         return self.get_layers()
 
+    def add_layer(self, layer):
+        """
+        """
+        if type(layer) is not Layer:
```
Comment (SAngeliuk) [inline NeoML/Python/neoml/Dnn/Dnn.py]: if not isinstance(layer, Layer) ?

File: NeoML/Python/neoml/Dnn/Dnn.py [hunk 3]
```diff
@@ -138,6 +138,21 @@ def layers(self):
...
+            raise ValueError('The `layer` is expected to be neoml.Dnn.Layer`')
+        self._add_layer(layer.internal)
+    
+    def delete_layer(self, layer):
+        if type(layer) is str:
+            self._delete_layer(layer)
+        elif type(layer) is Layer:
```
Comment (SAngeliuk) [inline NeoML/Python/neoml/Dnn/Dnn.py]: if not isinstance(layer, Layer) ?

File: NeoML/Python/neoml/Blob.py (hunk 1 above)
Comment (FedyuninV) [reply NeoML/Python/neoml/Blob.py]: Fixed

File: NeoML/Python/neoml/Dnn/Dnn.py (hunk 2 above)
Comment (FedyuninV) [reply NeoML/Python/neoml/Dnn/Dnn.py]: Fixed

File: NeoML/Python/neoml/Dnn/Dnn.py (hunk 3 above)
Comment (FedyuninV) [reply NeoML/Python/neoml/Dnn/Dnn.py]: Fixed

**Review by SAngeliuk — APPROVED** [review]

**Pull request merged**